src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/NavMenu.CallsMicrosoftGraph.razor

```diff
@@ -1,33 +1,33 @@
 ﻿<div class="top-row pl-4 navbar navbar-dark">
     <a class="navbar-brand" href="">BlazorServerWeb-CSharp</a>
-    <button class="navbar-toggler" @onclick="ToggleNavMenu">
+    <button title="Navigation menu" class="navbar-toggler" @onclick="ToggleNavMenu">
```

Comment:
Given that this button contains no text content, a screen reader can't say what it does. It just says "button", which isn't super helpful. This `title` attribute actually communicates something useful and is what the screen reader will announce.